src/arch/riscv/pagetable_walker.cc

```diff
@@ -490,8 +490,29 @@ Walker::WalkerState::recvPacket(PacketPtr pkt)
             Addr paddr = walker->tlb->translateWithTLB(vaddr, satp.asid, mode);
             req->setPaddr(paddr);
             walker->pma->check(req);
+
```

Comment:
This file changed permissions. This should (almost) never happen.

Reply:
Sorry, I am not sure how to check that in GitHub if file permission is changed?

Reply:
![Screenshot_20210319_074336](https://user-images.githubusercontent.com/109596/111797949-d4ac7800-8886-11eb-9e9b-324d07ddb8f3.png)

Do you see where it says "100644->100755"? That's saying the permissions changed. 

I think you can use `git ls-files` to see it in your local repo.